docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -137,8 +137,66 @@ Verify that after creation, a forward ACL rule can be removed and packets matchi
 - Verify that the results of "show acl rule {rule_name}" are empty for both rule names
 
 - Verify that packets with IPs matching the removed forwarding rules are dropped on this specific port
+-
+### Test Case # 7 - DHCP Forwarding Rule
 
-### Test Case # 7 - Scale test of ACL Table, add large amount of forward and drop rules
+#### Test Objective
+
+Verify that GCU is capable of adding blanket DHCP forwarding rules, and that traffic follows these rules accordingly
+
+#### Testing Steps
+
+- Use GCU to create blanket DHCP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that for both rules created, "show acl rule DYNAMIC_ACL_TABLE {rule_name} matches expected output
+
+- Verify that all operations were successful
+
+- Run pre-existing DHCP_Relay test with ACL rules applied and packets sent over the port that is blocked
+
+- Verify that pre-existing DHCP_Relay test is successful with ACL applied
+
+### Test Case # 8 - DHCPv6 Forwarding Rule
+
+#### Test Objective
+
+Verify that GCU is capable of adding blanket DHCPv6 forwarding rules, and that traffic follows these rules accordingly
+
+#### Testing Steps
+
+- Use GCU to create blanket DHCPv6 packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that for both rules created, "show acl rule DYNAMIC_ACL_TABLE {rule_name} matches expected output
+
+- Verify that all operations were successful
+
+- Run pre-existing DHCPv6_Relay test with ACL rules applied and packets sent over the port that is blocked
+
+- Verify that pre-existing DHCPv6_Relay test is successful with ACL applied
+
+### Test Case # 9 - ARP Forwarding Rule
+
+#### Test Objective
+
+Test that GCU is capable of adding a blanket ARP forwarding rule, and that proper ARP responses are sent while other packets are dropped
+
+#### Testing Steps
+
+- Use GCU to create a blanket ARP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that all operations were successful
+
+- Verify that ARP reply is received from server when ARP packet is sent on this port
```

Comment:
Please clarify the IP is configured to the ptf port, otherwise there will be no ARP reply.